### PR TITLE
chore: storybook migrate segment, timepicker, select, toast

### PR DESCRIPTION
**Pull request by anuraghazra**

### CODE REVIEW

File: src/segment/stories/Segment.stories.tsx [hunk 1]
```diff
@@ -1,40 +1,49 @@
...
+        codesandbox: DEFAULT_REACT_CODESANDBOX(["renderless-components@alpha"]),
+      },
+      {
+        tab: "React",
+        template: appTemplate,
+        language: "tsx",
+        copy: true,
+        codesandbox: DEFAULT_REACT_CODESANDBOX(["renderless-components@alpha"]),
+      },
+      {
```
Comment (anuraghazra) [inline src/segment/stories/Segment.stories.tsx:28]: we will use the createPreviewTabs helper after merging #136

Comment (navin-moorthy) [reply src/segment/stories/Segment.stories.tsx:28]: 👍

File: scripts/create-preview-tabs.ts [hunk 2]
```diff
@@ -46,3 +47,30 @@ export function createPreviewTabs(props: Props) {
...
+  return {
+      framework: "${ReactTab.toLowerCase()}",
+      files: {
+          "src/App.${type}": previews["${ReactTab}"][0],
+          "src/styles.css": previews["CSS"] ? previews["CSS"][0] : "",
+          "src/ToastUtils.component.${type}": previews["${utilTab}"] ? previews["${utilTab}"][0] : "",
+      },
+      userDependencies: ${joinStrs(deps)},
+  };`) as CodeSandboxTemplate;
+};
```
Comment (anuraghazra) [inline scripts/create-preview-tabs.ts:76]: Tried using normal higher order function but does not seem to be working. throwing `type` not defined error. i think internally we loose the function context and i think thats why addon-preview also uses string template literal functions

Comment (navin-moorthy) [reply scripts/create-preview-tabs.ts:76]: Oh okay okay !!

**Review by navin-moorthy — APPROVED** [review]
LGTM 🎉

**Comment by navin-moorthy** [comment]
Changes looks good, we can merge it after the #139 I think including those changes as well.

**Pull request merged**